scripts/cbsa/parse-metro-areas.ts

```diff
@@ -80,14 +80,19 @@ function formatGroup(groupItems: CsvRow[]) {
 
   const totalPopulation = sum(counties.map(county => county?.population || 0));
 
+  const nameForFormatUrl =
+    firstItem.cbsaTitle === 'New York-Newark-Jersey City, NY-NJ-PA'
+      ? 'New York City-Newark-Jersey City, NY-NJ-PA'
+      : firstItem.cbsaTitle;
```

Comment:
Curious if we considered doing this in the original CSV file? (scripts/cbsa/mar2020.csv)

Or is the CSV file in a pristine state from some external source and we want to keep it that way?

Reply:
do you mean updating the json file directly? updating the csv would mean updating a lot of instances of 'New York'. all in all i just figured i should update via the script to be safe, but can do it another way!

Reply:
Oh, hrm. I don't know how the metros data works well enough to know which json file you're referring to.  But yeah, I'm basically wondering if we want to do these sorts of fix-ups via patches in code or actually patch our underlying data file.  

It sounds like it's easier to do it in code.  That seems fine, though I could imagine we'll have more of these in the future so we might want to eventually make the code more general-purpose (have a METRO_RENAMES mapping dictionary or something). But we can start with this and evolve it as necessary